swagger.yaml

```diff
@@ -5586,6 +5588,8 @@ definitions:
       - 'dns-wildcard-not-configured'
       - 'disk-encryption-requirements-satisfied'
       - 'non-overlapping-subnets'
+      - 'vsphere-host-should-have-disk-uuid-enabled'
```

Comment:
```suggestion
      - 'platform-requirements-satisfied'
```

Reply:
Why not pass the UI a dedicated code? It is possible to have a generic platform error and a specific message, but text messages are difficult to parse

Reply:
Actually, I agree.  We should aim to make vsphere a first-class citizen, unlike add-ons like operators.  Let's shorten it to vsphere-disk-uuid-enabled?